<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Evopedia</class>
 <widget class="QMainWindow" name="Evopedia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evopedia Search</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="listView">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="searchField"/>
       </item>
       <item>
        <widget class="QComboBox" name="languageChooser">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuNetwork">
     <property name="title">
      <string>Network</string>
     </property>
     <addaction name="actionAuto"/>
     <addaction name="actionAllow"/>
     <addaction name="actionDeny"/>
    </widget>
    <addaction name="actionConfigure_Dumps"/>
    <addaction name="actionMap"/>
    <addaction name="menuNetwork"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionConfigure_Dumps">
   <property name="text">
    <string>Archives</string>
   </property>
  </action>
  <action name="actionMap">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Network Auto</string>
   </property>
  </action>
  <action name="actionAllow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Allow</string>
   </property>
  </action>
  <action name="actionDeny">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deny</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
